RileyLinkKit/UI/RileyLinkDeviceTableViewController.swift

```diff
@@ -82,14 +80,18 @@ public class RileyLinkDeviceTableViewController: UITableViewController, TextFiel
         if appeared {
             tableView.reloadData()
         }
+        
+        rssiFetchTimer = NSTimer.scheduledTimerWithTimeInterval(3, target: self, selector: #selector(updateRSSI), userInfo: nil, repeats: true)
```

Comment:
But `viewWillDisappear` should be balanced with `viewWillAppear`, so let's set `rssiFetchTimer` to `nil` in that method.